backend/src/backend/digesters/raw.py

```diff
@@ -0,0 +1,56 @@
+import base64
+from urllib.parse import urlparse
+from xml.etree.ElementTree import Element, SubElement, tostring
+
+from backend.models import KIND_ILLUSTRATION, Book
+
+
+async def gen_raw_digester() -> bytes:
+
+    lib_elem = Element("Library")
```

Comment:
I think it's not necessary to use etree for the whole library:
- it will require keeping a full, growing copy of the tree in memory while you iterate and then double it to convert to string
- it's probably not efficient on a super large list.
- it's not mandatory as what's outside the `<book />` nodes is ridiculous.

We should thus keep it for book and just fill-up an xml string